lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3727,6 +3727,13 @@ namespace {
       return nullptr;
     }
 
+    static bool isClangNamespace(const DeclContext *dc) {
+      if (const auto *ed = dc->getSelfEnumDecl())
+        return isa<clang::NamedDecl>(ed->getClangDecl());
```

Comment:
Would this trigger if `dc` is a C++ enum that got imported as a Swift enum?

Reply:
Wow, this is an autocomplete error. This supposed to check for `clang::NamespaceDecl` instead of `clang::Namedecl`. I hit tab too early.

Reply:
Now it should not.